### PR TITLE
Add more details on the meeting minutes

### PR DESCRIPTION
**Pull request by matteodelabre**
Thanks for the meeting notes @dixonary. I did not notice any significant difference between your notes and mine. This PR adds some extra details which I think should be recorded.

* Add a `.md` extension to enable Markdown formatting.
* Add header with meeting metadata.
* Add information about meeting participants.
* Add notes on the decision process.
* Add information about the next meeting time.

### CODE REVIEW

File: discussion-2020-01-11.md [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+</tr>
+</table>
+
+## Participants
+
+* @Eeems ([Oxide](https://github.com/Eeems/oxide) author).
+* @LinusCDE ([Toltec](https://github.com/toltec-dev/toltec) maintainer).
+* @dixonary ([Draft](https://github.com/dixonary/draft-reMarkable) launcher and .draft format author).
+* @matteodelabre ([Toltec](https://github.com/toltec-dev/toltec) maintainer).
+* @raisjn ([remux](https://rmkit.dev/apps/remux) author).
```
Comment (Eeems) [inline discussion-2020-01-11.md:18]: The usernames don't auto link :(

Comment (matteodelabre) [reply discussion-2020-01-11.md]: That’s strange, I expected them to. Anyway, I added the links manually. Thanks!

**Comment by Eeems** [comment]
Looks good to me

**Comment by dixonary** [comment]
Ah, missing .md file extension! That was why it's not formatting. Thanks!

**Pull request merged**